.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,40 @@
+name: Upload Go Unit tests
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Clean Install of Node version 18
```

Comment:
Can't we leverage any of those workflows:
- https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
- https://github.com/marketplace/actions/set-up-node-using-nvm